packages/core/src/guards.ts

```diff
@@ -115,8 +115,8 @@ export function stateIn<
   any // TODO: recheck if we could replace this with something better here
 > {
   function stateIn(
-    args: GuardArgs<TContext, TExpressionEvent>,
-    params: TParams
+    _args: GuardArgs<TContext, TExpressionEvent>,
+    _params: TParams
```

Comment:
those might be seen by users when debugging in devtools, I think it would be best to avoid `_` for those